### PR TITLE
put prospectus on a page

### PR DESCRIPTION
**Pull request by evykassirer**
addresses #38 and #37

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ navigation:
     desc: See who's speaking
   - title: Sponsors
     url: /sponsors/
-    desc: Cha-ching
```
Comment (evykassirer) [inline _config.yml]: :money_mouth_face: 💰

Comment (arshiamufti) [inline _config.yml]: good call on this lol

**Comment by arshiamufti** [comment]
:shipit:

**Comment by arshiamufti** [comment]
coolio! edit the description of the PR to include the other issue it addresses.

:shipit:!

**Pull request merged**